test/integration/auditingIntegrationTests.coffee

```diff
@@ -28,3 +40,65 @@ describe "Auditing Integration Tests", ->
 
         # async test :(
         setTimeout checkAudits, 1000
+
+  describe "TLS Audit Server", ->
+
+    it "should send TLS audit messages and save (valid)", (done) ->
+
+      options =
+        cert: fs.readFileSync "test/resources/trust-tls/cert1.pem"
+        key:  fs.readFileSync "test/resources/trust-tls/key1.pem"
+        ca: [ fs.readFileSync "test/resources/server-tls/cert.pem" ]
+
+      normalSocket = new (net.Socket)
+      client = new (tls.TLSSocket)(normalSocket)
+      client.connect 5051, 'localhost', options, ->
```

Comment:
Also, check that are ever get into this callback without an error. If you do we can rule out incorrect certificates.